requirements/requirements-test.txt

```
@@ -1,3 +1,4 @@
+ase==3.21.1
```

`requirements-test.txt` should pin to the _newest_ available version of a package.
```suggestion
ase==3.22.0
```

---

the newest version of this package caused CI issues, there was some discussion about it in a resolved thread above. Not sure if the version in this file needs to match what's in `setup.py` or not.

---

Higher versions then 3.21.1 cause pip to freak out in the CI because of the numpy dependency inconsistency.

---

@tommy-waltmann @DomFijan I think something got lost in translation here. We have two sets of testing requirements. One set (in `.circleci/ci-oldest-reqs.txt`) is for Python 3.6 and tests our _oldest_ supported dependencies. The other set of dependencies (in `requirements-test.txt`) is for Python 3.7, 3.8, and 3.9 and tests against the _newest_ released versions. Dependabot keeps the "newest" pinnings updated so we know if new package versions break our tests.

The confusion over NumPy versions shouldn't apply here. In the oldest requirements, I pinned to `ase==3.16.0`, which is a fairly old version where the xrdebye code was updated (according to the ase changelog). That version of `ase` doesn't require any particular version of NumPy and appears to work just fine in my local testing with the "oldest" environment.

In the newest dependencies, I pinned to `ase==3.22.0`. This requires `numpy>=1.15.0`. However, we are testing with NumPy 1.21.2, which means this should be fine.